doc/ref_model/chapters/chapter01.md

```diff
@@ -38,18 +38,18 @@ This section introduces the high-level principles of infrastructure abstraction
    - Networking resources (Limited to connectivity services only)
    - Acceleration resources
 1. Cloud Infrastructure exposed resources should be supplier independent
-1. All Cloud Infrastructure Application Programming Interfaces (APIs) must ensure Interoperability (multi-vendor, components substitution), drive Simplification, and open source implementations that have an open governance model (e.g. come from Open Communities or Standards Development Organisations). Through such APIs will cloud infrastructure resources be discovered/monitored by management entities, configured on behalf of VNFs and consumed by VNFs.
-1. VNFs should be modular and be designed to utilise the minimum resources required for the service
+1. All Cloud Infrastructure Application Programming Interfaces (APIs) must ensure Interoperability (multi-vendor, components substitution), drive Simplification, and open source implementations that have an open governance model (e.g. come from Open Communities or Standards Development Organisations). Through such APIs will cloud infrastructure resources be discovered/monitored by management entities, configured on behalf of workloads and consumed by workloads.
+1. Workloads should be modular and be designed to utilise the minimum resources required for the service
 1. Cloud Infrastructure shall support pre-defined and parameterised sizes
    - These pre-defined sizes will evolve over time
 1. Cloud Infrastructure provides certain resources, capabilities and features, and workloads should only consume these resources, capabilities and features
-1. VNFs that are designed to take advantage of Cloud Infrastructure accelerations shall still be able to run without these accelerations, however with the understanding that there will be potential performance impacts
+1. Workloads that are designed to take advantage of Cloud Infrastructure accelerations shall still be able to run without these accelerations, however with the understanding that there will be potential performance impacts
 1. Workloads shall not require hardware-dependent software
-   - This is in support of workload abstraction, enabling portability across the Infra and simplification of workload design
+   - This is in support of workload abstraction, enabling portability across the Cloud Infrastructure and simplification of workload design
    - This pertains to features that expose hardware directly to workloads, such as PCIe PassThrough (PCI-PT) and capabilities that use these features, for example, SR-IOV
    - Use of critical features in this category are governed by policies in the RM Appendix and referenced in RM Chapter 4
-1. Specific internal hardware details shall not be exposed above the Infra+VIM layers
-   - This is in support of workload abstraction, enabling portability across the Infra and simplification of workload design
+1. Specific internal hardware details shall not be exposed above the Cloud Infrastructure and Cloud Infrastructure Manager layers
```

Comment:
The layers are not defined and so what does "above" refer to?

Reply:
This part has been removed from the chapter.